docs/userEquitiesDescription.json

```diff
@@ -0,0 +1,67 @@
+index -> userLogin
+type -> user
+
+{
+	"email": "user@example.com",
+	"password": "XXXXX", //To be hashed
+	
+	"tokens": [
+		"token1",
```

Comment:
You mean access tokens ?

Reply:
Tokens will containt access tokens yep (but in a next iteration doing the login components, etc). Now our scope is to design the schema for multiple users, but using always the same user.